ui/vite.config.ts

```diff
@@ -2,5 +2,17 @@ import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
-	plugins: [sveltekit()]
+	plugins: [sveltekit()],
+	// Define the proxy configuration to communicate with the server.
+	server: {
+		proxy: {
+			'/api': {
+				// FIXME: Add Production endpoint for server
+				target: process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : '',
```

Comment:
❗ Need that hosting url

Reply:
What URL should I be hitting when doing local dev? Right now I'm on localhost:5173.

Reply:
That's the address to view the server (localhost:5173) to view the svelte site. This points to the port @grevian setup in his PR. I haven't really looked at how the server side is run though, so these ports will need to be adjusted as necessary.:loading: